ydb/core/protos/flat_scheme_op.proto

```diff
@@ -387,7 +389,9 @@ message TTableDescription {
     repeated TSequenceDescription Sequences = 39;
 
     optional TTableReplicationConfig ReplicationConfig = 40;
+
     optional bool Temporary = 41;
```

Comment:
Also there is problem that temporary tables must be ignored by backups. Somehow.